### PR TITLE
frontend Changes

### PR DESCRIPTION
**Pull request by sananand007**

### CODE REVIEW

File: service_flask.py [hunk 1]
```diff
@@ -30,6 +39,7 @@ def autocomplete():
     search_result = server.search(term)
     return render_template("search_results.html", term=term, results=search_result)
 
+
```
Comment (weihesdlegend) [inline service_flask.py]: remove this extra line

File: templates/welcome_page.html [hunk 2]
```diff
@@ -1,18 +1,82 @@
...
+    <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
 </head>
 
 <body>
 
-<h1>    Welcome to use the auto-complete system</h1>
+</html>
```
Comment (weihesdlegend) [inline templates/welcome_page.html]: I think this end tag should be put in the end?

File: templates/welcome_page.html [hunk 3]
```diff
@@ -1,18 +1,82 @@
...
+    var time = new Date();
+    var i = document.createElement("img");
+    i.src = "/getTime?time=" + time;
+    document.getElementById("time").innerHTML = time;
+}
+</script>
+<button onclick = "sendTimeToServer()">update time</button>
+<div id = "time">Time: </div>
+
+</body>
```
Comment (weihesdlegend) [inline templates/welcome_page.html]: please add a blank line for this file

File: service_flask.py (hunk 1 above)
Comment (sananand007) [reply service_flask.py]: done

File: templates/welcome_page.html (hunk 2 above)
Comment (sananand007) [reply templates/welcome_page.html]: done

File: templates/welcome_page.html (hunk 3 above)
Comment (sananand007) [reply templates/welcome_page.html]: done

**Pull request merged**